projects/ngx-time-scheduler/src/lib/ngx-time-scheduler.model.ts

```diff
@@ -17,6 +17,10 @@ export class Item {
   sectionID: number;
   tooltip?: string;
   metadata?: any;
+  resource: string;
+  supplier: string;
+  venue: string;
+  client: string;
```

Comment:
these should be user definable, not hard coded.
currently in our use case, we make the resource become the sectionID
Maybe the sectionID should allow a single ID or a map of ID's with keys being the user definable section name, and the value being the ID.

remember this also needs to still work as it currently works as well as with this extension.